roles/blacklight_app/tasks/main.yml

```diff
@@ -1,5 +1,11 @@
 ---
-- name: blacklight_app | Create the cron job
+- name: Blacklight_app | Install cron
```

Comment:
Could we change the base image to include cron, so this change is no needed?  Or move it to rails role, because most rails apps assume cron